ttt/lib/ttt/board.ex

```diff
@@ -1,10 +1,14 @@
 defmodule TTT.Board do
   def empty_board(dimension) do
-   Enum.to_list(1..dimension*dimension)
+    Enum.chunk(Enum.to_list(1..dimension*dimension), dimension)
   end
 
   def play_move(board, move, mark) do
-    List.replace_at(board, move - 1, mark)
+    Enum.map(Enum.with_index(board), fn({row, row_index }) ->
```

Comment:
Consider computing the `row_index` and `col_index` from move.
You can get there by using `rem(..)` and `div(...)`. Talk to Nathan about it.